docs/src/terminology.md

```diff
@@ -130,13 +134,15 @@ A [public key](terminology.md#public-key) and corresponding [private key](termin
 
 A fractional [native token](terminology.md#native-token) with the value of 0.000000001 [sol](terminology.md#sol).
 
+The smallest unit of the [native token](terminology.md#native-token), SOL. Equal to 0.000000001 [SOL](terminology.md#sol). Named after computer scientist [Leslie Lamport.](http://www.lamport.org/)
```

Comment:
Very nice, but can you delete the original definition too?

Reply:
yes, of course. An oversight.